components/engine_rocks/src/engine.rs

```diff
@@ -81,6 +82,11 @@ impl KvEngine for RocksEngine {
     fn cf_names(&self) -> Vec<&str> {
         self.0.cf_names()
     }
+
+    fn bad_downcast<T: 'static>(&self) -> &T {
```

Comment:
When this method will be used?

Reply:
It is currently used to get an Arc<DB> from RocksEngine in order to use the various `TableProperties` APIs. That is only until there is an abstracted version of them.